lib/ex_ach/batch/entry/fields/dfi_account_number.ex

```diff
@@ -0,0 +1,12 @@
+defmodule ExAch.Batch.Entry.Fields.DfiAccountNumber do
+  @moduledoc """
+  Receiver's account number at their financial institution. Left justify
+  """
+
+  @type t :: %__MODULE__{}
+  use ExAch.Field,
+    specifications: [
+      type: :alpha_num_string,
```

Comment:
@frahugo / @bokay seems like `alpha_numeric` would be an appropriate `type` name here.